### DISCUSSION TITLE
Allow date / datetime / time selection in a repeater

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13377] [Feature Requests]
There's a good choice of options in the repeater but it's missing time-related data type and interface. I'm assuming because the value has to be stored as a string.

Time and dates can be parsed from a string so it should certainly be possible?

**Comment by rijkvanzanten** [comment]
> There's a good choice of options in the repeater but it's missing time-related data type and interface. 

The repeater does have those types, right?

<img width="984" alt="CleanShot 2022-05-24 at 14 58 07@2x" src="https://user-images.githubusercontent.com/9141017/170111893-0d268757-8b3e-4176-b714-4e90014fa7ef.png">

**Reply by sxsws** [reply]
I'm so sorry! All this time and I didn't know see the options list was scrollable!
